docs/admin.rst

```diff
@@ -1,9 +1,71 @@
 Administrator Guide
 =====================
 
-At this point, you (the administrator) should have your own username and
-password, plus two-factor authentication through either the Google
-Authenticator app on your smartphone or a YubiKey.
+You (the administrator) should have your own username and password, plus
+two-factor authentication through either the Google Authenticator app on
+your smartphone or a YubiKey.
+
+.. _Responsibilities:
+
+Responsibilities
+----------------
+
+The SecureDrop architecture contains multiple hardened servers, and while we have
+automated many of the installation and maintenance tasks, a skilled Linux
+administrator and some manual intervention is required to responsibly run the system.
+
+This section outlines the tasks the administrator is responsible for in order to
+ensure that the SecureDrop server continues to be a safe place for sources to
+talk to journalists.
+
+Keep your SecureDrop Server Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should maintain awareness of SecureDrop updates and take any required
+manual action if requested in the `SecureDrop Release Blog`_. We recommend
+subscribing to the `SecureDrop RSS Feed`_ to stay apprised of new updates.
+
+Most often, the SecureDrop server will automatically update via apt. However,
+occasionally you will need to run the Ansible playbooks. We will inform you in
+the release blog when this is the case. If you are onboarded to our support portal,
+we will let you know in advance of major releases if manual intervention will be
+required.
+
+.. _`SecureDrop Release Blog`: https://securedrop.org/news
+.. _`SecureDrop RSS Feed`: https://securedrop.org/news/feed
+
+Keep your Network Firewall Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Given all traffic first hits the network firewall as it faces the non-Tor public
+network, you will want to ensure that critical security patches are applied.
+
+Be informed of potential updates to your network firewall. If you're using the
+suggested network firewall by FPF you can subscribe to the `Netgate RSS Feed`_
+to be alerted when releases occur. If critical security updates need to be
+applied, you can do so through the firewall's pfSense WebGUI.
+
+.. _`Netgate RSS Feed`: https://www.netgate.com/feed.xml
+
+Keep your Tails Drives Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should apply updates to your Tails drives as they are released, as they
+can contain critical security fixes. Subscribe to the `Tails RSS Feed`_ to be
+alerted of new releases. The online Tails drives once booted and connected to Tor
```

Comment:
Comma separation might help readability here: "The online Tails drives, once booted and connected to Tor, will alert you if upgrades are available."